### PR TITLE
Add bar lights frontend

### PR DESCRIPTION
**Pull request by ide-1**

### CODE REVIEW

File: frontend/src/App.css [hunk 1]
```diff
@@ -128,7 +154,7 @@ button:focus {
 }
 
 .BackButton:hover {
-  background-color: #AAAAAA;
+  background-color: #aaaaaa;
```
Comment (ViddeM) [inline frontend/src/App.css:157]: Hmmm 🤔

File: frontend/src/elements/Main.jsx [hunk 2]
```diff
@@ -33,8 +36,8 @@ export function Main() {
             } else {
                 let bigX = -1
                 let bigY = -1
-
-                setLights(response.response.data.lights.map(obj => {
+                console.log(response.response.data)
```
Comment (ViddeM) [inline frontend/src/elements/Main.jsx:39]: Plz remove console logs before merge :)

**Review by ViddeM — APPROVED** [review]
Not taken a look at the files changed in the backend PR but I assume that they're the same as they are there. 

Plz run the frontend through some autoformatter such as prettier or similar.

Also why change the hex rbg colors to lowercase? :P 

Otherwise looking good :+1:

File: frontend/src/App.css (hunk 1 above)
Comment (ide-1) [reply frontend/src/App.css:157]: did not change that on purpose very sus

**Pull request merged**